assets/src/domain/eventEditor/ui/tickets/ticketPriceCalculator/fields/usePrice.tsx

```diff
@@ -0,0 +1,26 @@
+import { useCallback } from 'react';
+
+import { useDataState } from '../data';
+import { BaseFieldProps } from './types';
+
+type BFP = BaseFieldProps;
+
+const usePrice = ({ field, price }: any): any => {
```

Comment:
Must need types here, it's not clear what `price` is, is it amount (number) or price object?

Reply:
sure, I was trying to make it work and forgot about proper types